### DISCUSSION TITLE
What's the differences between QT6 and 5 and standard version?

### DISCUSSION THREAD

**Discussion by InstaZDLL** [c0re100/qBittorrent-Enhanced-Edition#345] [Q&A] [answered]
Please tell me what's the differences between these:

qbittorrent_enhanced_4.4.1.10_Qt5_x64      44.1 MB
qbittorrent_enhanced_4.4.1.10_Qt6_x64      52.9 MB
qbittorrent_enhanced_4.4.1.10                     40.2 MB

![Opera Snapshot_2022-03-12_234117_github com](https://user-images.githubusercontent.com/72951793/158037535-efd51177-c36e-49f7-a345-39e0fbbd3790.png)

**Comment by LarLei** [comment, marked as answer]
qbittorrent_enhanced_4.4.1.10 40.2 MB = 32-bit

https://www.qt.io/blog/qt-6.2-vs.-qt-5.15-the-feature-parity-comparison
